spec/rails_helper.rb

```diff
@@ -0,0 +1,64 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```

Comment:
where is this file from?
remove it! always make sure your not committing things that do not belong to your feature